pkgs/top-level/all-packages.nix

```
@@ -22662,6 +22662,8 @@ in
 
   ht = callPackage ../applications/editors/ht { };
 
+  ht-rs = callPackage ../tools/networking/ht-rust { };
```

I think it's good to have all of `pname`, attribute name, and directory name agree if possible.
```suggestion
  ht-rs = callPackage ../tools/networking/ht-rs { };
```